commitlint.config.js

```diff
@@ -12,7 +12,7 @@ const Configuration = {
     'subject-full-stop': [2, 'never', '.'],
     'type-case': [2, 'always', 'lower-case'],
     'type-empty': [2, 'never'],
-    'type-enum': [
+    'type-enum': [2, 'always', [
```

Comment:
Also don't change eslint config file

Reply:
I think this lint config is invalid. cause commitlint keep yelling at me about value of `type-enum` is wrong.

Reply:
@lili21 this PR please send to `webpack-default` repo

Reply:
looks like `webpack-default` have nothing to do with `commitlint.config.js`

Reply:
@lili21 https://github.com/webpack-contrib/webpack-defaults/tree/next/templates

Reply:
Got it.